### PR TITLE
remove inotify-tools dep and AoLogs scaffolded for ansible based-operator

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**

- The `inotify-tools` as a dependency of Ansible based-operator images which was deprecated and it will no longer scaffold the `/bin/ao-logs` which was using it to print the Ansible logs in the side-car since the side-car ansible container was removed in the previous versions. [#2852](https://github.com/operator-framework/operator-sdk/pull/2852) 

**Motivation for the change:**

- PR to ensure that all required logs will be print in the operator container:
https://github.com/operator-framework/operator-sdk/pull/2580

- PR to remove the Ansible sidecar container: https://github.com/operator-framework/operator-sdk/pull/2586

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Unreleased
+
+### Removed
+
+- The `inotify-tools` as dependency of Ansible based-operator images which was deprecated and it will no longer scaffold the `/bin/ao-logs` which was using it to print the Ansible logs in the side-car since the side-car ansible container was removed in the previous versions. [#2852](https://github.com/operator-framework/operator-sdk/pull/2852)
```
Comment (joelanford) [inline CHANGELOG.md:5]: Revert this CHANGELOG change and replace with a fragment file in `changelog/fragments`

File: changelog/fragments/2852-pr.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: The `inotify-tools` as a dependency of Ansible based-operator images which was deprecated and it will no longer scaffold the `/bin/ao-logs` which was using it to print the Ansible logs in the side-car since the side-car ansible container was removed in the previous versions. [#2852](https://github.com/operator-framework/operator-sdk/pull/2852)
```
Comment (joelanford) [inline changelog/fragments/2852-pr.yaml:4]: Remove the PR link. That will be generated and appended automatically.

File: changelog/fragments/2852-pr.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: The `inotify-tools` dependency was removed and `/bin/ao-logs` are no longer generated for Ansible based-operators
```
Comment (joelanford) [inline changelog/fragments/2852-pr.yaml:30]: Nit: not a huge deal, but this is probably a little long for a header and might not render super well in the migration guide.

Not sure what the best general approach is for this. We could add a header character limit that we check when validating as part of the sanity test. WDYT?

File: changelog/fragments/2852-pr.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      body: >
+        The `inotify-tools` as a dependency of Ansible based-operator images which was deprecated in the previous versions was removed. This dependency was used to output the Ansible logs in a side-car container called `ansible`, which has no longer been scaffolded by default. In this way, if you have any customization using this dependency be aware that it is no longer in the Ansible based-operator image and then, feel free to remove this file `/bin/ao-logs` for your projects since it has no longer usage.  For further information also see the section `Remove Ansible container sidecar` in this guide.
```
Comment (joelanford) [inline changelog/fragments/2852-pr.yaml:32]: > For further information also see the section `Remove Ansible container sidecar` in this guide.

We're splitting out the migration guide into separate files for each release, so this text will end up in `v0.18.0.md`. I think we're also planning to split the existing migration guide into individual release migration guides, so linking over to that file might not be the best idea either.

Is there a PR related to removing the ansible container sidecar we could link to instead?

Comment (camilamacedo86) [reply changelog/fragments/2852-pr.yaml:32]: For the users know the steps, I think that the best would be the migration guide section and not the PR. I will try to figure out something here.

Comment (joelanford) [reply changelog/fragments/2852-pr.yaml:32]: If this isn't urgent, I can try to throw together a PR to split out the existing version upgrade guide, get that merged, and then you can link to the new one?

Comment (camilamacedo86) [reply changelog/fragments/2852-pr.yaml]: It has not the links. I mean it has just the information and then it was updated based on this comment. 
Could you please check and let us know if you are ok with?

**Pull request merged**